Added image support

spj2401Dev opened the pull request:
Hi, 
For a game with friends, I wanted to extend the game a bit to be able to include pictures, to have a braughter catalogue of clues. Now you can specify an image source for each clue.

A clue with an image will now look something like this

![image](https://github.com/user-attachments/assets/2befc80f-1783-437b-83bf-b4746d193cf1)

I appreciate your efforts on this project, and perhaps this is something worth adding to the project.

spj2401Dev committed:
+ Added image support

spj2401Dev committed:
Merge branch 'main' into main

sophxreg-cmd commented:
hey! this looks awesome. would it be possible to add images to the solution too?

spj2401Dev commented:
How about some sort of "game bundle" which is a zip file which includes the .json and the images. Then I can load them either externally or load them from this game bundel. Let me go implement that real quick

spj2401Dev committed:
Added ability to keep images locally via 'Game Bundels'

spj2401Dev commented:
So just pushed changes which now allow you to create as mentioned "Game Bundels". Basically you put your game.json and a images folder into a zip archive and upload. In your game.json you can access files in that Game Bundle via the `$` operator. So you can do `$images/Picture1.jpg`. 
I added it to my fork and added it to this PR, but you will probably need to grab this off my fork since this project seems pretty dead.

sophxreg-cmd commented:
thank you!!
